src/camera.cpp

```diff
@@ -2858,6 +2870,56 @@ bool Camera::get_demoMode() {
 	return appSettings.value("camera/demoMode", false).toBool();
 }
 
+
```

Comment:
We still have an ongoing issue with spaces vs. tabs.

Reply:
Should be able to settle it by importing your qt creator code styles .xml file. Please send it over when you get a chance, thanks!